src/fragmentarium/ui/display/Transliteration.tsx

```diff
@@ -150,6 +151,60 @@ function DisplayLine({
   )
 }
 
+function DisplayGenericControlLines({
+  line,
+  container = 'div'
+}: {
+  line: Line
+  container?: string
+}): JSX.Element {
+  const controlLine = line as ControlLines
+  const element = `Transliteration__GenericControlLine`
+  if (
+    controlLine.type === 'ImageDollarLine' ||
+    controlLine.type === 'LooseDollarLine'
```

Comment:
Why put the logic for selecting the component in two places? There is already `lineComponents`.